src/js/core/checkLastLottoNumberInput.js

```diff
@@ -0,0 +1,85 @@
+import { LOTTO_INFO } from '../constants/constant.js';
+import {
+  IS_OVERLAP_LOTTO_NUMBER_ERROR,
+  NOT_NUMBER_IN_RANGE,
+  NOT_NUMBER_TYPE_ERROR,
+  NOT_POSITIVE_NUMBER_ERROR,
+} from '../constants/errorMessage.js';
+import {
+  isPositiveNumber,
+  isValueTypeNumber,
+} from '../modules/checkInputValue.js';
+
+export const getLastLottoNumbers = () => {
+  const lastLottoNumbers = [];
+  document
+    .querySelectorAll('.last-lotto-winning-number-input')
+    .forEach(lastLottoNum => {
+      lastLottoNumbers.push(Number(lastLottoNum.value));
+    });
+  return lastLottoNumbers;
+};
+
+export const checkLastLottoNumberValid = () => {
+  const lastLottoNumbers = getLastLottoNumbers();
+  try {
+    let isLottoNumberInputPositive = false;
+    let isLottoNumberInRange = false;
+    if (checkLastLottoNumbersType(lastLottoNumbers)) {
+      isLottoNumberInputPositive =
+        checkLastLottoNumbersPositive(lastLottoNumbers);
+    }
+    if (isLottoNumberInputPositive) {
+      isLottoNumberInRange = checkLastLottoNumbersInRange(lastLottoNumbers);
+    }
+    if (isLottoNumberInRange) {
+      checkLastLottoNumbersOverlap(lastLottoNumbers);
+    }
+    return lastLottoNumbers;
+  } catch (err) {
+    window.alert('지난 로또 번호 입력창에는 ' + err.message);
+  }
+};
+
+export const checkLastLottoNumbersType = lastLottoNumbers => {
+  if (
+    lastLottoNumbers.filter(num => {
+      if (isValueTypeNumber(num)) return num;
```

Comment:
boolean 값을 기대하기 때문에 바로 리턴해주어도 될 거 같습니다.
```suggestion
      return isValueTypeNumber(num);
```